src/Microsoft.AspNet.Http.Abstractions/QueryString.cs

```diff
@@ -148,7 +148,7 @@ public static QueryString Create(IEnumerable<KeyValuePair<string, string>> param
         /// </summary>
         /// <param name="parameters"></param>
         /// <returns>The resulting QueryString</returns>
-        public static QueryString Create(IEnumerable<KeyValuePair<string, StringValues>> parameters)
+        public static QueryString Create(IQueryCollection parameters)
```

Comment:
Ok then to put this back and add 3rd overload which takes for `IQueryCollection` as it is non-boxing?